site/en/docs/devtools/css/reference/index.md

```diff
@@ -121,6 +124,48 @@ If your browser supports the `lab()` function, the element is green, otherwise i
 **Note**: At the time of writing, only Safari [supports the CIELAB color space](https://caniuse.com/?search=lab).
 {% endAside %}
 
+### View cascade layers {: #cascade-layers}
+
+[Cascade layers](/blog/cascade-layers/) enable more explicit control of your CSS files to prevent style-specificity conflicts. This is useful for large codebases, design systems, and when managing third-party styles in applications.
+
+To view cascade layers, [inspect](/docs/devtools/open/#elements) the element below and open **Elements** > **Styles**.
+
+<div class="cascade-box"></div>
```

Comment:
@devnook  @jeffposnick at the moment, we embeded demo in d.c.c like this sometimes. Do we have other recommend way to do it? (How about the web.dev code component?)

In the old WebFu, there's an iframe shortcode which we can used to separate the demo code.

Reply:
d.c.c. supports `Glitch`, `Codepen`, and a generic `IFrame` shortcodes for embedded content. Though for this sort of example—where you're asking folks to inspect an on-page element—it might be a better user experience to keep the HTML + CSS inline. I could imagine the Codepen/Glitch hosted examples adding in a bunch of extra DOM elements and styles that might distract from what you want the user to focus on.